api/tests/tooling/domain-builder/factory/certification/configuration/build-sco-blocked-access-date.js

```diff
@@ -0,0 +1,8 @@
+import { ScoBlockedAccessDate } from '../../../../../../src/certification/configuration/domain/read-models/ScoBlockedAccessDate.js';
+
+export const buildScoBlockedAccessDate = function ({ scoOrganizationType, reopeningDate } = {}) {
```

Comment:
Mettre des valeurs par defaut comme dans les autres builders ?

```suggestion
export const buildScoBlockedAccessDate = function ({ scoOrganizationType = 'COLLEGE', reopeningDate = 'XXXXXX' } = {}) {
```

Reply:
Je peux, mais comme à chaque fois que je l'utilise je crée les deux entrées à chaque fois, je trouve que ce qui aurait plus d’intérêt de renvoyer la liste des 2 ScoBlockedAccessDate (sinon on repasse les paramètres systématiquement donc on se sert jamais de ceux par défaut)

Reply:
J'ai finalement créé un builder collège et un lycée avec des valeurs par défaut